apps/damap-frontend/src/app/components/landing-page/landing-page.component.css

```diff
@@ -0,0 +1,224 @@
+.background {
+  background: url("../../../assets/landing-page-background.png");
+  background-color: var(--accent-color-very-light);
+  background-repeat: no-repeat;
+  background-position: right top;
+  height: 100%;
+  width: 100%;
+}
+
+.container {
+  padding-right: 6%;
+  padding-left: 6%;
+  max-width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+  scroll-behavior: smooth;
+}
+
+.logo {
+  margin-top: 15px;
+  padding-right: 15px;
+  padding-left: 15px;
+  height: 60px;
+  width: 300px;
+}
+
+.info-bar-typography {
+  font: var(--body-large);
+  color: var(--landing-page-info-bar-color);
+}
+
+.info-bar {
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+.logo-info-bar {
+  /* padding-top: 80px; */
+  margin-top: auto;
+}
+
+.landing-page-graphic-container {
+  width: 100%;
+  margin-top: -30px;
+  margin-left: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.landing-page-graphic {
+  margin: 0 auto;
+  max-width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.double-items {
+  display: flex;
+  align-items: center;
+  margin-bottom: auto;
+}
+
+.left-side-container {
+  width: 100%;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-bottom: 15px;
+}
+
+.title-typography {
```

Comment:
**Headline**: Please set the view for 13-inch screens to use display-medium styling.

I tested this, and using display-medium works well for screens around 1600px wide, which includes many 13-inch resolutions. You can define it as follows:

````
.responsive-title {
  font: var(--display-large);
  ...
}

  @media (max-width: 1600px) {
    font: var(--display-medium);
    ....
  }
  
  you can define that in **styles.scss** as:
    --display-medium: #{mat.get-theme-typography(
      $app-light-theme,
      display-medium,
      font
    )};
}

```